backend/backend.csproj

```diff
@@ -4,6 +4,9 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DocumentationFile>bin\Debug\netcoreapp3.1\backend.xml</DocumentationFile>
```

Comment:
For the reviewers: this compile flag ensures that xml documentation is generated!